server/node.go

```diff
@@ -249,6 +291,8 @@ func (n *Node) initStores(clock *hlc.Clock, engines []engine.Engine) error {
 
 	// Bootstrap any uninitialized stores asynchronously.
 	if bootstraps.Len() > 0 {
+		// If no NodeID has been assigned yet, make sure this happens first.
```

Comment:
s/make sure this happens first/allocate a new node ID by supplying 0 to initNodeID/